src/azure-cli/setup.py

```diff
@@ -115,7 +115,7 @@
     'azure-mgmt-redis~=13.1.0',
     'azure-mgmt-relay~=0.1.0',
     'azure-mgmt-reservations==2.0.0',  # TODO: Use requirements.txt instead of '==' #9781
-    'azure-mgmt-resource==21.1.0b1',
+    'azure-mgmt-resource==21.1.0',
```

Comment:
Please also upgrade the package version of `azure-mgmt-resource` in the following configuration file:
- src/azure-cli/requirements.py3.Darwin.txt
- src/azure-cli/requirements.py3.Linux.txt
- src/azure-cli/requirements.py3.windows.txt